app/controllers/apo_controller.rb

```diff
@@ -87,8 +87,16 @@ def set_apo_metadata apo, md_info
     apo.metadata_source      = md_info[:metadata_source]
     apo.agreement            = md_info[:agreement].to_s
     apo.default_workflow     = md_info[:workflow ] unless (!md_info[:workflow] || md_info[:workflow].length < 5)
-    apo.use_license          = md_info[:use_license]
     apo.default_rights       = md_info[:default_object_rights]
+    if md_info[:use_license].blank?
+      # delete use licenses, ensure defaultObjectRights datastream is saved correctly
+      apo.defaultObjectRights.creative_commons = nil
+      apo.defaultObjectRights.open_data_commons = nil
+      apo.defaultObjectRights.creative_commons_human = nil
```

Comment:
thanks for the feedback. i think the next step is that i will make a PR for `dor-services` that allows `use_license=` to set a null license if `nil` is passed in. then i will change this PR to use that method.